requirements.txt

```diff
@@ -1,3 +1,3 @@
 selenium >= 4.0.0
-robotframework >= 3.2.2
-robotframework-pythonlibcore >= 2.2.1
+robotframework >= 4.1.3
```

Comment:
Is there are hard requirement for version increase? Or is this a preferred/suggested one?

Reply:
Sorry, sometimes I am a bit forgetful even of things/changes I have made myself. Yes, we had bumped up the tested version in the matrix dropping out version 3.2.2 due to the many issues that were occurring and due to the fact we are currently on rf 6.x.

One question I have which I want to ask, if not to myself, is why raise the minimum to 4.1.3 and not 4.1.0 or more simply >4.x? Really I am asking what are difference amongst the point releases of the core? Are they security ones such that it makes sense to jump to the latest? or are they a feature releases that giving a lower version number support a widest audience - which as I type this I hear Tatu telling just the latest.

Reply:
3.2.2 was dropped because it has problems with pythonlibcore 4, I didn't test plc 3 though.


There's no specific reason for 4.1.3, I just picked the latest 4.x and that's also what we test.
I checked just now and there are some [small backwards incompatibilities between 4.0.x and 4.1.x](https://github.com/robotframework/robotframework/blob/master/doc/releasenotes/rf-4.1.rst#id38). However, I think upgrading shouldn't be too difficult if some people have stuff that depends on the earlier behaviour. Also, there are some nice bug fixes and features in 4.1.x series. 4.1.1 adds official support for Python 3.10 and 4.1.2 has some improvements to memory use. 4.1.3 just fixes a regression introduced in 4.1.2. Overall I think users would benefit from running 4.1.3 compared to other 4.x versions.

Reply:
I'm good with 4.1.3. For anyone interested [here was a discussion](https://robotframework.slack.com/archives/C0K0240NL/p1672429142861239) we had about versions within the Robot Framework Slack community.